frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/geometry-helper.ts

```diff
@@ -0,0 +1,71 @@
+import type { Position, Size, UUID } from 'digital-fuesim-manv-shared';
+import type { Feature } from 'ol';
+import type { Coordinate } from 'ol/coordinate';
+import type { Geometry } from 'ol/geom';
+
+export interface PositionableElement {
+    readonly id: UUID;
+    readonly position: Position;
+}
```

Comment:
It might be handy if the respective models would implement this so we get typescript errors if we break this contract

Reply:
Nevermind, the way you're using this type ensures the constraint is fulfilled.